<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painel | Sistema de Controle de Comandas</string>
  </property>
  <widget class="QLabel" name="label_dashboardTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sistema de Controle de Comandas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_dashboard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_dashboard">
     <item>
      <widget class="QPushButton" name="pushButton_itemManagement">
       <property name="font">
        <font>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Gestão de itens/produtos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_userManagement">
       <property name="text">
        <string>Gestão de usuários</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clientManagement">
       <property name="font">
        <font>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Gestão de clientes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_orderPaperManagement">
       <property name="text">
        <string>Gestão de comandas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_report">
       <property name="font">
        <font>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Relatório</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_systemExit">
       <property name="text">
        <string>Sair do sistema</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>20</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_dashboardLoggedAsUser">
    <item>
     <widget class="QLabel" name="label_loggedAsUser">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Logado como:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_loggedAsUser">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
